clippy_lints/src/manual_unwrap_or_default.rs

```diff
@@ -124,15 +125,14 @@ fn handle_match<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'tcx>) -> bool {
         // We now check the `None` arm is calling a method equivalent to `Default::default`.
         && let body_none = body_none.peel_blocks()
         && is_default_equivalent(cx, body_none)
-        && let Some(match_expr_snippet) = snippet_opt(cx, match_expr.span)
     {
         span_lint_and_sugg(
             cx,
             MANUAL_UNWRAP_OR_DEFAULT,
             expr.span,
             "match can be simplified with `.unwrap_or_default()`",
             "replace it with",
-            format!("{match_expr_snippet}.unwrap_or_default()"),
+            format!("{}.unwrap_or_default()", Sugg::hir(cx, match_expr, "..").maybe_par()),
             Applicability::MachineApplicable,
```

Comment:
We don't normally use `MachineApplicable` on suggestions that might return `".."`.

Reply:
Good point! Fixed.